wordpress-mobile/WordPress-Android
# IA create content bottom sheet changes

Partial fix for #11013 

This PR does the following:

- Changes the copy of the actions in the main bottom sheet
- Adds a title to the bottom sheet

Below a comparison for before/after bottom sheet:


| Before | After |
|---|---|
| ![image](https://user-images.githubusercontent.com/47797566/72115108-32216c80-3346-11ea-860c-2567d74fadaf.png) | ![image](https://user-images.githubusercontent.com/47797566/72169953-56299000-33d0-11ea-8945-1c72eeab3db8.png) |

## To test
- Compile and run `vanilla flavor`. Smoke test the app (switching between main sections in the bottom nav bar and using the My Site features). Check no evident abnormalities are triggered.

- Compile and run the `zalpha flavor`
- Select the FAB and check that a bottom sheet is opened and is similar to the After image above. The Editor is opened when selecting the Blog Post/Site Page action from the bottom sheet. 
- Check that you can create a new Post/Page and it is Published/Drafted
- Check that no action is taken tapping on the bottom sheet `Add new` title

NOTE: RELEASE NOTES not modified yet since still bihind feature flag.

PR submission checklist:
